src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,44 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_PAIRS_NUMBER = 10;
+    private static int sizeCounter;
+    private Object[] keys;
+    private Object[] values;
+
+    public StorageImpl() {
+        keys = new Object[MAX_PAIRS_NUMBER];
+        values = new Object[MAX_PAIRS_NUMBER];
+        sizeCounter = 0;
+    }
```

Comment:
```suggestion
    public StorageImpl() {
        keys = new Object[MAX_PAIRS_NUMBER];
        values = new Object[MAX_PAIRS_NUMBER];
    }
```

size will be initialized by 0 by default

Reply:
removed fm Constructor